src/workspacer/workspacer.csproj

```diff
@@ -29,10 +29,16 @@
     <ProjectReference Include="..\workspacer.FocusIndicator\workspacer.FocusIndicator.csproj" ReferenceOutputAssembly="false" />
     <ProjectReference Include="..\workspacer.TitleBar\workspacer.TitleBar.csproj" ReferenceOutputAssembly="false" />
     <ProjectReference Include="..\workspacer.Watcher\workspacer.Watcher.csproj" ReferenceOutputAssembly="false" />
+	  <ProjectReference Include="..\workspacer.FocusBorder\workspacer.FocusBorder.csproj" ReferenceOutputAssembly="false" />
```

Comment:
Weird indentation here with a mixture of tabs and spaces. Keep to spaces only please :)